### PR TITLE
some configuration update

### PR DESCRIPTION
**Pull request by MeirShpilraien**

### CODE REVIEW

**Comment by MeirShpilraien** [comment]
@rafie if you can take a look at the get started documentation and let me know if its correct it will be great, thanks

File: docs/configuration.md [hunk 1]
```diff
@@ -36,6 +41,10 @@ Tells the python interpreter where to look for the default python libraries
 
 **[compiled dir]**/src/cpython/
```
Comment (rafie) [inline docs/configuration.md]: Maybe we should mention /opt/redislabs/lib/modules/python3 here.

File: docs/quickstart.md [hunk 2]
```diff
@@ -20,21 +20,22 @@ Install:
...
 If you running gears on the same machine on which it was compile then its enough just loading the RedisGears module:
 
 `--loamodule <path to redisgears.so`
 
-If you run RedisGears on another machine, some extra setup is needed. For Gears to run properly its need to have the cpython directory it was compiled with. All the required files are located in `/src/deps/cpython/` make sure to put this directory on the same machine where Redis running and use `PythonHomeDir` config variable to tell RedisGears where it should look for this cpython directory:
-
-`--loamodule <path to redisgears.so> PythonHomeDir <path to cpython direcotry>`
+If you run RedisGears on another machine, some extra setup is needed. For Gears to run properly its need to have the virtual environment which was created when it was compiled. All the required files are located in `/opt/redislabs/lib/modules/python3` make sure to put this directory on the same machine where Redis running and at the same path (i.e `/opt/redislabs/lib/modules/python3`).
 
 ## Tests
```
Comment (rafie) [inline docs/quickstart.md]: Tests are written in python using the [RLTest](https://github.com/RedisLabsModules/RLTest) library.

Simply run:
```
make test
```

File: docs/quickstart.md [hunk 3]
```diff
@@ -20,21 +20,22 @@ Install:
 
 ### Compile
```
Comment (rafie) [inline docs/quickstart.md]: ### Compile

Run:
```bash
make fetch # this will aquire git submodules
make all
```

File: docs/quickstart.md (hunk 2 above)
Comment (rafie) [reply docs/quickstart.md]: Also:
### Prerequisites
* Install [redis 5.0](https://redis.io/) on you machine.
* On Mac OSX (High Sierra), install Xcode command line tools:
```bash
xcode-select --install
```
* Then, run:
```bash
make setup
```

**Pull request merged**